app/gui/src/model/module.rs

```diff
@@ -39,6 +41,11 @@ pub use double_representation::tp::QualifiedName as TypeQualifiedName;
 #[derive(Debug, Clone, Copy, Fail)]
 #[fail(display = "Node with ID {} was not found in metadata.", _0)]
 pub struct NodeMetadataNotFound(pub ast::Id);
+/// Failure for missing node metadata.
```

Comment:
Duplicated docs. Additionally, in other places we do not add docs for error structures, as the message in `fail` should be descriptive enough.